suse2013/common/trim-verbatim.xsl

```diff
@@ -12,61 +12,97 @@
   xmlns:xsl="http://www.w3.org/1999/XSL/Transform" >
 
 <xsl:template match="programlisting/text()|screen/text()|synopsis/text()">
```

Comment:
Yes, I have thought about that (respectively, had thought about half a year ago). See lines 17, 32, 89 (the green versions)...

Now, I am not sure that this is the best way to do it, but it did not create problems afaics. If you would rather see that, I can also press most of the logic from lines 17/32/89 into the `match` attribute. This would avoid calling this template on unrelated nodes.

However, we would probably not save a lot of code. (Or any code at all.)